appeals/web/src/server/appeals/appeal-details/correct-appeal-type/__tests__/correct-appeal-type.test.js

```diff
@@ -0,0 +1,69 @@
+import { parseHtml } from '@pins/platform';
+import supertest from 'supertest';
+import { appealData, lpaQuestionnaireData } from '#testing/app/fixtures/referencedata.js';
+import { createTestEnvironment } from '#testing/index.js';
+import nock from 'nock';
+
+const { app, installMockApi, teardown } = createTestEnvironment();
+const request = supertest(app);
+const appealId = appealData.appealId;
+const lpaQuestionnaireId = appealData.lpaQuestionnaireId;
+const baseUrl = `/appeals-service/appeal-details/${appealId}/lpa-questionnaire/${lpaQuestionnaireId}`;
+
+describe('correct-appeal-type', () => {
+	beforeEach(installMockApi), afterEach(teardown);
+
+	describe('GET /change', () => {
+		it('should render the correctAppealType change page', async () => {
+			nock('http://test/')
+				.get(`/appeals/${appealId}/lpa-questionnaires/${lpaQuestionnaireId}`)
+				.reply(200, lpaQuestionnaireData);
+			const response = await request.get(`${baseUrl}/is-correct-appeal-type/change`);
+
+			const elementInnerHtml = parseHtml(response.text).innerHTML;
+
+			expect(elementInnerHtml).toMatchSnapshot();
+			expect(elementInnerHtml).toContain('Is the appeal type correct (LPA response)?</h1>');
+		});
+	});
+
+	describe('POST /change', () => {
+		it('should re-direct to LPA questionnaire if "yes"', async () => {
```

Comment:
This test description feels a little too brief, I think it should mention what is "yes" (i.e. the provided answer)

Reply:
I will look into test naming once we start getting a standard (currently on my list)